src/fsw/FCCode/OrbitController.cpp

```diff
@@ -18,16 +30,20 @@ OrbitController::OrbitController(StateFieldRegistry &r, unsigned int offset) :
     sched_valve1_f("orbit.control.valve1", Serializer<unsigned int>(1000)),
     sched_valve2_f("orbit.control.valve2", Serializer<unsigned int>(1000)),
     sched_valve3_f("orbit.control.valve3", Serializer<unsigned int>(1000)),
-    sched_valve4_f("orbit.control.valve4", Serializer<unsigned int>(1000))
+    sched_valve4_f("orbit.control.valve4", Serializer<unsigned int>(1000)),
+    J_ecef_f("orbit.control.J_ecef", Serializer<lin::Vector3d>(0,10,100))
+
 {
     add_writable_field(sched_valve1_f);
     add_writable_field(sched_valve2_f);
     add_writable_field(sched_valve3_f);
     add_writable_field(sched_valve4_f);
+    add_writable_field(J_ecef_f);
     sched_valve1_f.set(0);
     sched_valve2_f.set(0);
     sched_valve3_f.set(0);
     sched_valve4_f.set(0);
+    J_ecef_f.set(0.0);
```

Comment:
Perhaps you mean this? That being said, `lin` types are guaranteed to default construct full of zeros -- part of them being `constexpr` types.
```suggestion
    J_ecef_f.set(lin::zeros<lin::Vector3d>());
```